resources/lang/en/thoughts/alive/general.json

```
@@ -1668,7 +1668,7 @@
         "thoughts": [
             "Feels like there are ants pricking {PRONOUN/m_c/poss} pelt...",
             "Wonders if other cats laugh with {PRONOUN/m_c/object} or at {PRONOUN/m_c/object}...",
-            "Quietly debates fleeing from a conversation with a Clanmate who is talking about something they're incredibly incorrect about",
+            "Is debating fleeing a conversation with a Clanmate who is talking about something they clearly know nothing about",
```

```suggestion
            "Debates fleeing a conversation with a Clanmate who is talking about something they clearly know nothing about",
```
a little smoother when we aren't repeated two 'ing' words